.github/contributing.md

```diff
@@ -0,0 +1,38 @@
+# Contributing
+
+## Security
+
+If you think you've found a **security issue**, please do not mention it in this repository.
+Instead, email renovate-disclosure@whitesourcesoftware.com with as much details as possible so that it can be handled confidentially.
+
+## Bug Reports and Feature Requests
+
+If you've found a **bug** or have a **feature request** then please create an issue in this repository (but search first in case a similar issue already exists).
+
+## Code
+
+If you would like to fix a bug or implement a feature, please fork the repository and create a Pull Request.
+
+Before you start any Pull Request, it's recommended that you create an issue to discuss first if you have any doubts about requirement or implementation.
+That way you can be sure that the maintainer(s) agree on what to change and how, and you can hopefully get a quick merge afterwards.
+Also, let the maintainers know that you plan to work on a particular issue so that no one else starts any duplicate work.
+
+## Do not force push to your pull request branch
+
+Please do not force push to your PR's branch after you have created your PR, as doing so makes it harder for us to review your work.
+PRs will always be squashed by us when we merge your work.
+Commit as many times as you need in your pull request branch.
+
+## Re-requesting a review
+
+Please do not ping your reviewer(s) by mentioning them in a new comment.
+Instead, use the re-request review functionality.
+Read more about this in the [GitHub docs, Re-requesting a review](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#re-requesting-a-review).
+
+## Slack collaboration with maintainers
+
+Sometimes the codebase can be a challenge to navigate, especially for a first-time contributor.
+We don't want you spending an hour trying to work out something that would take us only a minute to explain.
+
+For that reason, we have a Slack channel dedicated to helping anyone who's working on or considering Pull Requests for Renovate.
+Please email rhys@arkins.net and simply mention that you need an invitation to the channel and you'll be added ASAP.
```

Comment:
Does the "get help on Slack" apply to this repo as well?

Reply:
Sure